src/server/mod.rs

```diff
@@ -40,11 +38,9 @@ pub use self::node::{create_raft_storage, Node};
 pub use self::resolve::{PdStoreAddrResolver, StoreAddrResolver};
 pub use self::raft_client::RaftClient;
 
-pub type CopStream<T> = Box<Stream<Item = T, Error = mpsc::SendError<T>> + Send>;
-
 pub enum OnResponse<T> {
-    Unary(Box<FnBox(T) + Send>),
-    Streaming(Box<FnBox(CopStream<T>, Option<CpuPool>) + Send>),
+    Unary(mpsc::Sender<T>),
```

Comment:
I think `oneshot::Sender` and `mpsc::Sender` have different performance. So I think we shouldn't mix them if we can.